src/argilla/server/schemas/v1/datasets.py

```diff
@@ -117,10 +118,12 @@ class DatasetCreate(BaseModel):
     workspace_id: UUID
 
 
-class DatasetUpdate(BaseModel):
+class DatasetUpdate(UpdateSchema):
     name: Optional[DatasetName]
     guidelines: Optional[DatasetGuidelines]
 
+    __non_explicit_none__ = {"name", "guidelines"}
```

Comment:
I think the guidelines are nullable in DB, so, we should let users remove the guidelines for a dataset.

Reply:
```suggestion
    __non_explicit_none__ = {"name"}
```